insertion_sort/insertion_sort.py

```diff
@@ -0,0 +1,20 @@
+def insertionSort(alist):
+   """
+    Performs a insertion Sort
+    :param alist: Iterable of elements   
+    """
+   for index in range(1,len(alist)):
+     currentvalue = alist[index]
+     position = index    
+     while position>0 and alist[position-1]>currentvalue:
+         alist[position]=alist[position-1]
```

Comment:
Use a consistent style.
You are assigning as `var=value` whereas the best way is `var = value` which you have already used at majority of other places.